python/mim_control/dg/wbc.py

```diff
@@ -0,0 +1,144 @@
+"""This file is a demo for using the DG whole body controller.
```

Comment:
About where to place the file: I wanted to place it first under `mim_control/dynamic_graph/wbc.py`. However, that's the same path that is used by the dynamic graph plugins. Therefore, I went with `dg` for now.

Should we place this kind of code in the `dynamic_graph_py` subfolder maybe?

Reply:
You should be able to put it in mim_control/dynamic_graph/wbc_graph.py

Reply:
Good points, I moved the files under `mim_control/dynamic_graph/wbc_graph.py`